### PR TITLE
Sentiment Lexicon

### PR DESCRIPTION
**Pull request by Sebastian-Mlr**
- Implemented Opinion Lexicon (optimized Accuracy ~58,91%)
- Implemented Vader Lexicon (Accuracy ~45,03%, still some room for Optimization here)

### CODE REVIEW

**Comment by mariushes** [comment]
@Sebastian-Mlr Sieht schon gut aus :)
Ein paar Anmerkungen noch:
- Das Sentiment nenne wir in unserem Fall anders: gut, ok, schlecht oder auch 0, 1, 2
- Die Benennung der Variablen mit den Daten in der zweiten Code Zelle ist etwas verwirrend mit dem Namen der Sentiment lexicons im Namen.
- Mir ist nicht ganz klar, was du in den jeweils letzten beiden zahlen des Abschnitts 2. und 3. (von dir auch 2. genannt) machst, eventuell kannst du dort nochmal kommentieren.
- Wo vergleichst du denn die Predections mit den tatsächlichen Werten um dementsprechend auch die Measures zu berechnen (Accuracy, Precision, Recall, f1). Du könntest hier `sklearn.metrics.classification_report` nutzen.
